scriptTag/script.css

```diff
@@ -0,0 +1,3 @@
+function fun1(){
```

Comment:
# File specific comment 
Please change file extension .css to .js

Reply:
done